test/e2e/json-rpc/switchEthereumChain.spec.js

```diff
@@ -21,11 +20,18 @@ describe('Switch Ethereum Chain for two dapps', function () {
           .withNetworkControllerDoubleGanache()
           .build(),
         dappOptions: { numberOfDapps: 2 },
-
-        localNodeOptions: {
-          ...defaultGanacheOptions,
-          concurrent: [{ port: 8546, chainId: 1338 }],
-        },
+        localNode: ['anvil', 'anvil'],
+        localNodeOptions: [
+          {},
+          {
+            blockTime: 2,
+            vmErrorsOnRPCResponse: false,
+            mnemonic:
+              'phrase upgrade clock rough situate wedding elder clever doctor stamp excess tent',
+            port: 8546,
+            chainId: 1338,
+          },
+        ],
```

Comment:
now we can pass how many clients we want and the corresponding options using arrays.
This will allow us to use combinations of anvil + bitcoin, or anvil + solana in the future

Reply:
What do you think about only having one property: `localNode` which would contain the options, plus a new field that identifies the type ("ganache" or "anvil")?
So:

```typescript
localNode: [
  "anvil", // if there are no options we can allow just the string `"anvil"` or `"ganache"`
  {
    type: "anvil",
    blockTime: 2,
    // ...
  }
]
```

Reply:
thank you that's a great suggestion :heart_eyes: pushing the changes!
Just one small comment: given that in all the tests we are using `localNodeOptions`, I will set the name of this variable `localNodeOptions` so this way we don't need to change all the specs (which might be ~200 files) in this PR.
But we can rename it in a subsequent PR

Reply:
I think `localNodeOptions` is fine :-)